robot-server/robot_server/sessions/router.py

```diff
@@ -169,44 +167,49 @@ async def remove_session_by_id(
 
 
 @sessions_router.post(
-    path="/sessions/{sessionId}/inputs",
-    summary="Create a session control input.",
+    path="/sessions/{sessionId}/controls",
+    summary="Create a session control command.",
```

Comment:
Nit, but maybe:

```suggestion
    summary="Issue a control command to a session",
```

(Also note the lack of period. We haven't discussed this as a team yet, but [here's why I don't think this should be punctuated](https://opentrons.atlassian.net/wiki/spaces/RPDO/pages/594280793/FastAPI+and+pydantic+best+practices#:~:text=in%20the%20documentation%2C%20summary%20will%20be%20rendered%20as%20a%20heading%2C%20menu%20item%2C%20or%20table-of-contents%20entry.).

Reply:
~~Totally separate thing:~~

~~Seeing "control command" in context over and over is making it feel a little redundant to me. Doesn't every command "control"? Should we call it just `session command`? (Not necessarily suggesting that for this PR.)~~ Looks like we're going with "actions," per another discussion.